src/Item/Presentation/Requests/Express/ItemRequestCriteria.ts

```diff
@@ -11,6 +11,7 @@ class ItemRequestCriteria extends RequestCriteria implements ICriteria
 {
     constructor(query: ParsedQs, url: string)
     {
+        // @ts-ignore
```

Comment:
???

Reply:
this is because the  update of shaExp
![image](https://user-images.githubusercontent.com/38164371/132257901-f1aadd03-0663-43a0-9e22-2eca9e979170.png)

the update needs to be validated

Reply:
So need to fix before merge